### PR TITLE
Added achievement progress on right-click remove

### PR DESCRIPTION
**Pull request by CornerPin**

### CODE REVIEW

File: garrysmod/gamemodes/sandbox/entities/weapons/gmod_tool/stools/remover.lua [hunk 1]
```diff
@@ -79,6 +79,8 @@ function TOOL:RightClick( trace )
 
 	end
 
+	self:GetOwner():SendLua( string.format( "for i=1,%i do achievements.Remover()end", count ) )
```
Comment (fruitwasp) [inline garrysmod/gamemodes/sandbox/entities/weapons/gmod_tool/stools/remover.lua]: `Count`

Comment (CornerPin) [inline garrysmod/gamemodes/sandbox/entities/weapons/gmod_tool/stools/remover.lua]: Oops

**Comment by Kefta** [comment]
Wouldn't it be better to send a net message instead of a constant string?

**Comment by CornerPin** [comment]
Then i have to do this for LeftClick too. I don't know is it necessary to add new net message name for this little purpose, and these functions is not called so often.

**Comment by Kefta** [comment]
I didn't see the other SendLua call. I guess it could be fixed after this is merged.

**Pull request merged**